test/extended/testdata/build-secrets/s2i-binary-dir/Dockerfile

```diff
@@ -0,0 +1,9 @@
+USER default
+EXPOSE 8080
+ENV RACK_ENV production
+ENV RAILS_ENV production
+COPY . /opt/app-root/src/
+RUN scl enable rh-ruby22 "bundle install"
+CMD ["scl", "enable", "rh-ruby22", "./run.sh"]
+
+USER default
```

Comment:
That was a holdover from the original testcase input. Not needed in this private testdata area. I've removed the file.